app/views/admin/companies/edit.html.erb

```diff
@@ -8,11 +21,13 @@
     <%= f.number_field :hire_count, in: 0..500, step: 1, value: @company.hire_count %><br>
     <%= f.label(:hire_count, "Number of people hiring") %>
   </div>
-  <div class="form-group">
-    <%= f.hidden_field :status %>
-  </div>
-    <%= f.submit "Submit" %>
+  <%= f.hidden_field :status %>
+  <%= f.hidden_field :name %>
+  <%= f.hidden_field :title %>
+  <%= f.hidden_field :email %>
```

Comment:
I'm not sure if we want to give admins that ability.  Let's discuss this as a group because I'm not sure.